README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+## Diana Reed
+
+### now
+Hi, I'm Diana Reed, a **MAPSS** student in Education and Society. A former Boston public school teacher, I'm now trying to "homeschool" my own two CPS kids, a 4th grader and an 8th grader, while finishing my thesis. We'll see how this goes!
```

Comment:
Good luck with the kids at home. I've got three kids (4 year old, 2 year old, and 4 month old). Fortunately my wife stays at home full-time with them. I cannot imagine a full course/workload and having to take care of my kids as well.

Reply:
Best practice to leave a blank line between headers and other content. Like this

```
### now

Hi, I'm Diana Reed, a **MAPSS** student in Education and Society. A former Boston public school teacher, I'm now trying to "homeschool" my own two CPS kids, a 4th grader and an 8th grader, while finishing my thesis. We'll see how this goes!
```

It will help prevent formatting issues in the future